packages/esm-patient-list-app/src/patient-table/patient-table.component.tsx

```diff
@@ -67,7 +67,7 @@ const PatientTable: React.FC<PatientTableProps> = ({
     () =>
       patients.map((patient, index) => {
         const row = {
-          id: index,
+          id: patient.uuid,
```

Comment:
This needs to be member uuid, instead of patient uuid.
Thanks!

Reply:
@vasharma05 Thanks for the review. I have fixed the proposed changes.